pykickstart/commands/realm.py

```diff
@@ -57,7 +58,8 @@ def _parseJoin(self, args):
                                                        "membership-software=",
                                                        "one-time-password=",
                                                        "no-password",
-                                                       "computer-ou="))
+                                                       "computer-ou=",
+                                                       "computer-name="))
```

Comment:
This should not be added to the F19 class.

Reply:
Whoops!!! Thanks.